src/test/resources/json/error-file-invalid-rows-unknown.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
Should this file be renamed to: `error-file-validation-error-invalid-rows-unknown`?